pisa/core/distribution_maker.py

```diff
@@ -87,17 +99,71 @@ def __init__(self, pipelines, label=None):
             if not isinstance(pipeline, Pipeline):
                 pipeline = Pipeline(pipeline)
             self._pipelines.append(pipeline)
+
+        data_run_livetime = None
+        if set_livetime_from_data:
+            #
+            # Get livetime metadata if defined in any stage in any pipeline
+            #
+            for pipeline_idx, pipeline in enumerate(pipelines):
```

Comment:
somehow, pipeline is not an instance of the Pipeline class anymore at this point...

Reply:
My bad. Since I define `__iter__` as iterating over the pipelines contained, this should simply be:

```python
for pipeline_idx, pipeline in enumerate(self):
    ...
```